Automattic/pocket-casts-ios
# End of Year: epilogue story design

| 📘 Project: #376 |
|:---:|

Add the design for the epilogue story.

| iPhone 8 | iPhone 14 | iPhone 14 Pro Max |
| -------- | ---------- | ------------------- |
| ![Simulator Screen Shot - iPhone 8 - 2022-10-27 at 09 40 41](https://user-images.githubusercontent.com/7040243/198286863-6d187eae-7d4b-40e0-aff9-6d647cf01720.png) | ![Simulator Screen Shot - iPhone 14 - 2022-10-27 at 09 40 48](https://user-images.githubusercontent.com/7040243/198286893-8932c1a7-7e45-450b-821c-8ea514d4e5f8.png) | ![Simulator Screen Shot - iPhone 14 Pro Max - 2022-10-27 at 09 40 55](https://user-images.githubusercontent.com/7040243/198286923-89112ecf-f67f-4438-86e4-a22b302660ed.png) |

## To test

I recommend using these devices to test, given they provide different screen sizes:

* iPhone 8
* iPhone 14
* iPhone 14 Pro Max

1. Enable the `endOfYear` flag in `FeatureFlag.swift`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Navigate to the last story
4. ✅ Check the design
4. Tap "Replay"
4. ✅ Stories should replay

## Checklist
